lib/composer.js

```diff
@@ -229,9 +229,7 @@ export default class Composer extends Disposable {
         if (event.source.range) {
           message.range = [[event.source.range.start - 1, 0], [event.source.range.end - 1, Number.MAX_SAFE_INTEGER]]
         }
-      }
-
-      if (event.log) {
+      }      if (event.log) {
```

Comment:
This is why disabling the rule carte blanche is not an ideal solution. Looks like these extra spaces should be preceded with a newline.